README.md

```diff
@@ -50,13 +50,26 @@ NXST_FLOW reply (xid=0x4):
  cookie=0x0, duration=103.842s, table=0, n_packets=0, n_bytes=0, idle_age=103, priority=0 actions=NORMAL
 ```
 
-We have also extended Oko with the CLI command to update the BPF maps of the given BPF program. Once the filter prog
-is loaded, the below command can be used to write entry (key-value pair) to the given BPF program.
+We have also extended Oko with the CLI command to update the BPF maps of the given BPF program.
+To test this feature we have prepared simple `dropper.c` example. We assume that Oko handles traffic between hosts.
 
 ```
-ovs-ofctl update-map br0 1 0 key 1 0 0 0 value 1 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0
+clang -O2 -target bpf -c examples/bpf/dropper.c -o /tmp/dropper.o
+ovs-ofctl load-filter-prog br0 1 /tmp/dropper.o
+ovs-ofctl add-flow br0 priority=1,in_port=1,filter_prog=1,actions=output:2
+ovs-ofctl add-flow br0 priority=1,in_port=2,actions=output:1
+ovs-ofctl dump-flows br0
+NXST_FLOW reply (xid=0x4):
+ cookie=0x0, duration=103.730s, table=0, n_packets=0, n_bytes=0, idle_age=103, priority=1,in_port=1,filter_prog=1 actions=output:2
+ cookie=0x0, duration=103.842s, table=0, n_packets=0, n_bytes=0, idle_age=103, in_port=2,actions=output:1
 ```
 
+Now, the traffic (e.g. ping) should not be dropped. In order to drop the packets destined to some IP address
+(let's take `172.16.0.14` as example) update BPF map with appropriate IP address (the value `1` corresponds to DROP action):
+
+`ovs-ofctl update-map br0 1 0 key 172 16 0 14 value 1 0 0 0`
```

Comment:
I'd move this to the above code block with a shorter comment on top:
```bash
...
# Drop (value=1) packets destined to IP 172.16.0.14:
$ ovs-ofctl update-map br0 1 0 key 172 16 0 14 value 1 0 0 0
```

Reply:
Also, is it expected that 1 is `1 0 0 0` whereas 172.16.0.14 is `172 16 0 14`? Shouldn't it be `key 172 16 0 14 value 0 0 0 1` or `key 14 0 16 172 value 1 0 0 0`?

Reply:
You're right, the byte order of `iphdr->daddr` in the BPF program is different. I'm fixing it.